signature/mechanism_other.go

```diff
@@ -0,0 +1,13 @@
+//go:build !containers_image_openpgp && !containers_image_disable_signing && !cgo
+// +build !containers_image_openpgp,!containers_image_disable_signing,!cgo
+
+package signature
+
+// CgoIsDisabled indicates as a compiler error that this package requires cgo.
+//
+// The containers_image_openpgp build tag enables the OpenPGP implementation but
+// beware that it is not actively maintained and is considered insecure.
+//
+// The containers_image_disable_signing build tag disables signing and will
+// return an error when attempting to sign an image at runtime.
+var CgoIsDisabled = ContainersImageSignatureRequiresCgo
```

Comment:
RequiresCGoAndGPGMEDevelopmentHeaders , or maybe ContainersImageSignatureReadREADMEOfContainersImage ?

I guess I don’t really care to polish this that much…

Reply:
I've seen a few errors with this format - "RequresCgo" - in the past.

Main place I've seen a build failure enforce something about the version or go tags is here:

https://github.com/protocolbuffers/protobuf-go/blob/v1.28.1/runtime/protoimpl/version.go#L58

It used to generate something like `var ProtoVersionIsTooOld = undefined` but was recently changed to use Go's uint overflow checking to enforce the build failure.

Reply:
The pedantic point was that CGo is not the _only_ requirement. But this works well enough already.